src/pages/test-report/components/fault-counter/fault-counter.scss

```diff
@@ -0,0 +1,19 @@
+fault-counter {
+  >div {
```

Comment:
is the `>` a typo?

Reply:
I was targeting immediate child, but it's not strictly necessary, removed it.